### PR TITLE
Snaps

### PR DESCRIPTION
**Pull request by uuilmer**
So far was able to:

1. Implement Snaps which are photo messages that can only be seen one before they are deleted. Am planning to make the User experience better before merging.

Demonstration:

ezgif-6-e9c3910cae30

### CODE REVIEW

**Comment by uuilmer** [comment]
Was able to allow a User to click on a picture and show that picture in full-screen:

ezgif-6-8bab223fd858

**Comment by uuilmer** [comment]
Was able to add a functioning video feature. Users can now send and view Snap videos:

ezgif-4-f291de6d728b

File: app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java [hunk 1]
```diff
@@ -1,14 +1,21 @@
 package com.example.pix.chat.adapters;
 
 import android.content.Context;
-import android.util.DisplayMetrics;
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java:4]: Fix all the string literals in this file please.

File: app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java [hunk 2]
```diff
@@ -82,17 +100,109 @@ public void bind(int position, List<Message> messages) {
...
+
+                        // No more interaction with the full-screen ImageView
+                        imageContainer.setOnClickListener(null);
+                    });
+                });
+            }
+
+
+            // If this is a Snap content...
+            if (message.isSnap()) {
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java]: This part for handling the snap messages is pretty long. Can you make it a separate method?

**Review by xuyueee — COMMENTED** [review]
Looks good. Please address comments before merging.

**Pull request merged**